flocker/control/_protocol.py

```diff
@@ -137,12 +163,32 @@ def connectionLost(self, reason):
         self.control_amp_service.disconnected(self)
 
 
+DEPLOYMENT_CONFIG = Field(u"configuration", repr, u"The cluster configuration")
+CLUSTER_STATE = Field(u"state", repr, u"The cluster state.")
+
+LOG_SEND_CLUSTER_STATE = ActionType(
+    "flocker:controlservice:send_cluster_state",
+    [DEPLOYMENT_CONFIG, CLUSTER_STATE],
+    [],
+    "Sending the configuration and state of the cluster.")
+
+AGENT = Field(u"agent", repr, u"The agent we're sending to")
+
+LOG_SEND_TO_AGENT = ActionType(
+    "flocker:controlservice:send_state_to_agent",
+    [AGENT],
+    [],
+    "Send the configuration and state of the cluster to a specific agent.")
```

Comment:
The description for this action is in a different ... mood?  Or something.  As compared to the description for `LOG_SEND_CLUSTER_STATE` above.